main.js

```diff
@@ -0,0 +1,123 @@
+//make a search button that fetches the country data when you click it
+const searchButton = document.getElementById("search-button");
+
+searchButton.addEventListener('click', (event) => {
+    fetchData();
+});
+
+//make a search bar that fetches the country data when pressing enter
+const searchBar = document.getElementById('input-field');
+searchBar.addEventListener('keyup', setQuery)
+
+let query = '';
+
+function setQuery(e) {
+    query = e.target.value;
+    if (e.keyCode === 13) {
```

Comment:
Het gebruik van `.keyCode` is deprecated ([hier](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) te lezen). Gebruik daarentegen `.code`